vue/src/components/ModelRunDetail.vue

```diff
@@ -138,6 +138,14 @@ const determineDownload = () => {
           {{ modelRun.region || "(none)" }}
         </div>
       </v-row>
+      <v-row dense>
```

Comment:
I implemented this, but wasn't happy with the way I was checking if we're in the scoring app or not (basically, just a string comparison on the `ApiService.apiPrefix` static variable). I started refactoring the `ApiService` a bit to make checking that a bit cleaner, but I'll include that in a follow up since it touches things out of scope for this PR.

Reply:
Yeah, maybe being in scoring mode  checking should exist in the state store, because it is mostly used as a reactive property for templates.